Classes/Core/Testbase.php

```diff
@@ -598,6 +598,18 @@ public function dumpClassLoadingInformation()
         }
     }
 
+    /**
+     * Reload class loading information if available
+     *
+     * @return void
+     */
+    public function reloadClassLoadingInformation()
+    {
+        if (ClassLoadingInformation::isClassLoadingInformationAvailable()) {
```

Comment:
Hmm, this means, we reload ONLY if it exists, right? So it doesn't generate it "on the first run"?

Reply:
We can also move this to the RequestBootstrap class directly.

Reply:
> Hmm, this means, we reload ONLY if it exists, right? So it doesn't generate it "on the first run"?

Yes, this function only loads classes which were dumped already. In RequestBootstrap we can be sure to have dumped classes already so we only need to load them again.

> We can also move this to the RequestBootstrap class directly.

Sure, i can move it back to RequestBootstrap directly.

Reply:
I moved it back to RequestBootstrap.